NAMESPACE

```diff
@@ -47,23 +51,16 @@ export(get_ssm_by_regions)
 export(get_ssm_by_sample)
 export(get_ssm_by_samples)
 export(get_study_info)
-export(heatmap_mutation_frequency_bin)
 export(id_ease)
 export(liftover_bedpe)
 export(populate_each_tool_result)
 export(review_hotspots)
-export(setup_expreession_data)
-export(setup_fusions)
-export(setup_study)
-export(study_check)
 export(tidy_gene_expression)
 export(write_sample_set_hash)
-import(ComplexHeatmap)
 import(DBI)
 import(GAMBLR.data)
 import(GAMBLR.helpers)
 import(GAMBLR.utils)
-import(GAMBLR.viz)
```

Comment:
If we're getting rid of this dependency here we should also get rid of this in `DESCRIPTION` - duh why is this file has to be edited by hand and is not automated!!

Reply:
Right, This has been fixed. Thanks for catching it!